Content.Client/GameObjects/Components/Weapons/FlashableComponent.cs

```diff
@@ -22,7 +22,7 @@ public sealed class FlashableComponent : SharedFlashableComponent
         private CancellationTokenSource _cancelToken;
         private TimeSpan _startTime;
         private double _duration;
-        private FlashOverlay _overlay;
+        private FlashWhiteOverlay _overlay;
```

Comment:
Pretty sure FlashWhiteOverlay needs removing.

Reply:
In that case I think this could solve two problems at once (in regards to the need to remove MsgFlash)